O21.Game/Scenes/Button.fs

```diff
@@ -6,46 +6,53 @@ open Raylib_CsLo
 open O21.Game
 
 [<RequireQualifiedAccess>]
-type ButtonState = Default | Hover | Clicked
+type ButtonState =
+    | Default
+    | Hover
+    | Clicked
 
 type Button = {
     Font: Font
     Text: string
     Position: Vector2
     State: ButtonState
 } with
+
     static member DefaultColor = Raylib.GRAY
     static member HoverColor = Raylib.DARKGRAY
     static member ClickedColor = Raylib.BLACK
 
-    static member Create (font: Font) (text: string) (position: Vector2): Button = {
+    static member Create (font: Font) (text: string) (position: Vector2) : Button = {
         Font = font
         Text = text
         Position = position
         State = ButtonState.Default
     }
 
     member private this.Rectangle =
-        let size = Raylib.MeasureTextEx(this.Font, this.Text, float32 this.Font.baseSize, 1.0f)
+        let size =
+            Raylib.MeasureTextEx(this.Font, this.Text, float32 this.Font.baseSize, 1.0f)
+
         Rectangle(this.Position.X, this.Position.Y, size.X, size.Y)
 
-    member this.Render(): unit =
+    member this.Render() : unit =
```

Comment:
_(Note.)_ This is the only aspect of formatting that a little bit concerns me: I don't want a space here.

But I haven't found a way to remove it using Fantomas settings. So… let it be, I guess?